src/index.html

```diff
@@ -4,6 +4,7 @@
     <title>Fabric8 Stack Analysis</title>
     <meta charset="UTF-8">
     <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport">
+    <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly/patternfly.css" crossorigin="anonymous">
```

Comment:
You can add a comment saying that we just use css declaration from patternfly

Reply:
Done